docs/reference/pipeline-yaml-reference.md

```diff
@@ -2087,10 +2087,10 @@ YAML parser, which is not a Semaphore 2.0 feature but the way YAML files work.
 - [Machine Types](https://docs.semaphoreci.com/ci-cd-environment/machine-types/)
 
 [ubuntu1804]: https://docs.semaphoreci.com/ci-cd-environment/ubuntu-18.04-image/
-[macos-mojave-xcode11]: https://docs.semaphoreci.com/ci-cd-environment/macos-mojave-xcode-11-image/
+[macos-mojave-xcode11]: https://docs.semaphoreci.com/ci-cd-environment/macos-xcode-11-image/
```

Comment:
Same here, we should remove "mojave" from link test and leave "macos-xcode11"

Reply:
Thanks, great catches (what I've done is find / replace 🤦‍♀️) I'll correct this.